cli/spec/support/output_helpers.rb

```diff
@@ -52,23 +54,25 @@ module OutputHelpers
         line = 0
         @real.zip(@expected) do |real, expected|
           line += 1
-          error = values_match?(real.split(/\S+/), expected)
-          if error
-            @error ||= ""
-            @error << "\n#{error}"
-            @error.strip!
+          fields = real.split(/\s{2,}/)
```

Comment:
@kke is this a reasonable way to split the tty-table output fields? This just leaves out empty fields (`nil` or `''`), it would be nice to also have those included in `fields`.